### ISSUE TITLE
Studies with HTML tags, such as italicised species, are not filtering properly in the "My analyses" list

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/web-eda#780]
For example 
Study **SCORE Seasonal Transmission S. haematobium Cluster Randomized Trial**

Make a new analysis - name it or change a filter or something so it saves - then click "My analyses"
![image](https://user-images.githubusercontent.com/308639/146181609-1951f250-3d18-4e09-9f3f-b9314a02da1b.png)

The list will be empty, but it shouldn't be!

**Comment by SheenaTomko** [comment]
Just to add to this ticket, it seems when you click "My analyses", the search analyses field auto-populates with a version of the study name that contains the html tags. When you remove the html or clear the search bar, you see the analysis you made

<img width="867" alt="Screen Shot 2022-02-16 at 9 49 01 AM" src="https://user-images.githubusercontent.com/54597319/154302240-e27b9049-8099-46d0-b8c7-3aa6317a6e8d.png">


<img width="727" alt="Screen Shot 2022-02-16 at 9 49 08 AM" src="https://user-images.githubusercontent.com/54597319/154302422-d11707b8-0af9-40f9-8f35-e364a399e83e.png">

**Comment by bobular** [comment]
There's also an endless loop in the `AllAnalyses` component logic.  It's debounced I think, so doesn't slow down the browser perceptibly.  Will investigate with this ticket.

**Comment by aaronwlsong** [comment]
Looks good, closing ticket.

 if I click "My analyses" from dropdown workspace menu:
<img width="1876" alt="Screen Shot 2022-02-24 at 3 07 53 PM" src="https://user-images.githubusercontent.com/61077845/155599462-1e908844-f22b-4227-9674-d35980b1e519.png">


If I clicked the icon of "My analyses" on right top of website:
The previous studies won't be listed, if I want to see all previous studies I made, I need to click "x" to remove the auto-filled name for latest study, and then the all my previous studies will pop out.  

After I finish the last study, I clicked "My analyses":

<img width="1870" alt="Screen Shot 2022-02-24 at 2 43 08 PM" src="https://user-images.githubusercontent.com/61077845/155598900-52d4ffdc-df59-462f-b9c2-3fa0fcbdfc6d.png">

To see all my previous studies, I have to manually remove the latest study's name. 
<img width="1881" alt="Screen Shot 2022-02-24 at 2 43 41 PM" src="https://user-images.githubusercontent.com/61077845/155599134-32cf0086-deeb-4ae2-80a2-5f0c161ff2a2.png">
